addons/website_slides/models/slide_channel.py

```diff
@@ -468,6 +469,16 @@ def create(self, vals_list):
 
         return channels
 
+    def copy_data(self, default=None):
+        default = default or {}
+        if 'slide_ids' not in default:
+            default['slide_ids'] = [(0, 0, line.copy_data()[0]) for line in self.slide_ids]
+        if 'name' not in default:
+            default['name'] = f"{self.name} ({_('copy')})"
+
+        default['enroll'] = 'public'
```

Comment:
why not `if 'enroll' not in default` ?

Reply:
so it's more explicit to add "copy=False" on enroll, but we should ask LNA opinion to be sure

also about product, currently it will take the same product right ?